dissect/target/plugins/browsers/edge.py

```diff
@@ -32,18 +32,3 @@ class EdgePlugin(ChromiumMixin, Plugin):
     BrowserHistoryRecord = create_extended_descriptor([UserRecordDescriptorExtension])(
         "browser/edge/history", GENERIC_HISTORY_RECORD_FIELDS
     )
-
-    @export(record=BrowserDownloadRecord)
-    def downloads(self):
```

Comment:
Is this deletion intentional?

Reply:
This PR was still in draft mode. Unfortunately since you have started to react while it was in draft there is no means within GH anymore to specify that something is WIP, so please consider this a race-condition ;-)

Reply:
Are you sure you want to go down this road though? It looks like the reduction of LoC comes at the cost of additional complexity. In your opinion, is this worth the trade-off?

Reply:
> Are you sure you want to go down this road though? It looks like the reduction of LoC comes at the cost of additional complexity. In your opinion, is this worth the trade-off?

I'm not sure what this comment is in reply of exactly, can you elaborate?

Back on topic of this original comment, this deletion isn't a namespace function, but actual functionality of the Edge plugin, is it not? Or am I missing something?

Reply:
The code was deleted by accident and I put it back. It should have not been visible in the PR but I created a draft PR (not ready for review) to check the code myself.

The other comment was just a generic observation and has nothing to do with the deleted lines (that are now back).

Reply:
Ah my bad, GitHub seemed to still show it as deleted when I made that last comment.